kong/router/expressions.lua

```diff
@@ -19,8 +19,28 @@ local PROTOCOLS_OVERRIDE = {
 }
 
 
-local function get_exp_and_priority(route)
+-- net.port => net.dst.port
+local function verify_expression(route)
   local exp = route.expression
+
+  if not exp then
+    return nil
+  end
+
+  if exp:find("net.port", 1, true) then
+    ngx.log(ngx.WARN, "The field 'net.port' of expression is deprecated, " ..
+                      "please use 'net.dst.port' instead.")
+
+    return exp:gsub("net%.port", "net.dst.port")
```

Comment:
Could we use the returned value to determine if `net.port` is used?

Reply:
changed the code.